packages/sage-assets/lib/stylesheets/components/_search.scss

```
@@ -18,26 +18,23 @@ $-search-icon: "::before";
     z-index: 2;
     margin-left: sage-spacing(xs);
     margin-right: -1 * sage-spacing();
-    color: sage-color(grey, 500);
+    color: sage-color(charcoal, 400);
```

Looks like this value should be:

```suggestion
    color: sage-color(grey, 600);
```